.github/workflows/gpu-runner.yaml

```diff
@@ -1,6 +1,9 @@
 name: Self-Hosted Runner
 on:
   workflow_dispatch:
+  schedule:
+    # At 07:00 UTC daily.
+    - cron: "0 7 * * *"
```

Comment:
Do we want to run daily? I think we should be okay if we run weekly, but I don't have strong feelings about this. Just thinking about mitigating costs (even if low).

Reply:
I remember thinking the same thing, but John pointed out it doesn't cost much and it can be nice to track down when an upstream thing changes and only have to parse through a env diff between days instead of weeks.